### PR TITLE
Proguard cleanup

### LINKED ISSUE #363: Proguard setup issues
Address @wojtek-kalicinski's comments on proguard changes.

https://github.com/nickbutcher/plaid/pull/354#pullrequestreview-134016711

### PR DESCRIPTION
**Pull request by keyboardsurfer**
Fix #363
Address dangling comments of #354

### CODE REVIEW

File: about/build.gradle [hunk 1]
```diff
@@ -32,9 +32,7 @@ android {
     }
 
     buildTypes {
-        release {
-            proguardFiles 'proguard-rules.pro'
-        }
+        release { }
```
Comment (nickbutcher) [inline about/build.gradle]: Can we remove this empty block?

File: app/build.gradle [hunk 2]
```diff
@@ -41,7 +41,8 @@ android {
     buildTypes {
         release {
             minifyEnabled true
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'),
```
Comment (nickbutcher) [inline app/build.gradle]: Why wrap here?

File: about/build.gradle (hunk 1 above)
Comment (nickbutcher) [reply about/build.gradle]: Same throughout.

File: app/build.gradle (hunk 2 above)
Comment (keyboardsurfer) [reply app/build.gradle]: Line length

**Pull request merged**